pytest-server-fixtures/pytest_server_fixtures/xvfb.py

```diff
@@ -43,7 +43,10 @@ class XvfbServer(object):
 
     def __init__(self):
         tmpdir = mkdtemp(prefix='XvfbServer.', dir=Workspace.get_base_tempdir())
-        for servernum in range(os.getpid(), 65536):
+        pid_max = 65536
+        with open('/proc/sys/kernel/pid_max') as pid_max_file:
```

Comment:
Is this guaranteed to exist on all Linux distributions? We should probably safely fall back to 65536 if the file doesn't exist.

Reply:
We do fall back, it's set before we open the /proc file. That /proc file was added sometime in 2.5, so it should exist, but it may throw an exception if /proc isn't mounted -- but we probably have bigger problems in that case.